kyuubi-server/src/test/scala/org/apache/kyuubi/engine/spark/SparkProcessBuilderSuite.scala

```diff
@@ -300,10 +300,12 @@ class SparkProcessBuilderSuite extends KerberizedTestHelper with MockitoSugar {
   }
 
   test("[KYUUBI #5009] Test pass spark engine log path to spark conf") {
+    System.getenv().put("KYUUBI_WORK_DIR_ROOT", "/tmp/kyuubi/unit-test")
```

Comment:
env map should not be modifiable.

java.lang.ProcessEnvironment
```java
        theUnmodifiableEnvironment
            = Collections.unmodifiableMap
            (new StringEnvironment(theEnvironment));
```

Reply:
Thanks, i'd find other way.